src/monitor/node_active_protocol.c

```diff
@@ -965,7 +988,89 @@ stop_maintenance(PG_FUNCTION_ARGS)
 					  currentNode->nodePort,
 					  currentNode->pgsrSyncState,
 					  currentNode->reportedLSN,
+					  currentNode->candidatePriority,
+					  currentNode->replicationQuorum,
 					  message);
 
 	PG_RETURN_BOOL(true);
 }
+
+
+/*
+ * set_node_replication_settings sets node replication parameters.
+ * It could be called on both primary or standby nodes. The function
+ * returns true on success.
+ */
+Datum
+set_node_replication_settings(PG_FUNCTION_ARGS)
+{
+	int32 nodeid = PG_GETARG_INT32(0);
+	int candidatePriority = PG_GETARG_INT32(1);
+	bool replicationQuorum = PG_GETARG_BOOL(2);
+	AutoFailoverNode *currentNode = NULL;
+	bool nodeUpdated = false;
+
+
+	ereport(LOG, (errmsg("updating node replication properties for node %d, candidate_priority=%d, replication_quorum=%s",
+						  nodeid, candidatePriority, replicationQuorum?"true":"false")));
```

Comment:
This looks more like a debug message, but I'm not too sure. Do we want to keep it?